test/unit/test_store.rb

```diff
@@ -102,6 +102,22 @@ def test_create_missing_translations_in_top_level_file
     assert_nil translation.text
   end
 
+  def test_create_missing_keys_uses_replace_locale_in_path_transformation
+    calls = []
+    @store.define_singleton_method(:replace_locale_in_path) do |*args|
+      calls << args
+      "/tmp/special_path/en.yml"
```

Comment:
You're just testing that you've stubbed a method. No go. Can't we just call the method, and assert the outputted path? :smile: